.devcontainer/Dockerfile

```diff
@@ -10,13 +18,16 @@ RUN apt install -y postgresql-12 postgresql-client-12
 # Install chromium-driver
 RUN apt install -y chromium-driver
 
+# Install ruby (compiling takes some time)
+RUN curl -sSL https://rvm.io/mpapis.asc | gpg --import -
```

Comment:
Ja pouzivam rbenv ale rvm tiez asi je ok.

Reply:
Ja tiež, ale to sa mi nepodarilo do toho debian imagu správne nainštalovať.